src/ui/views/EditAwards.basketball.tsx

```diff
@@ -0,0 +1,329 @@
+					obj.setValue(obj.state.select.value);
+					return {
+						...prevState,
+						aws,
+					};
+				}
+				aws[type][numberPlayer] = {
+					pid: p.pid,
+					name: p.firstName + " ",
+					tid: p.tid,
+					abbrev: p.abbrev,
+					pts: p.currentStats == undefined ? 0.0 : p.currentStats.pts,
+					trb: p.currentStats == undefined ? 0.0 : p.currentStats.trb,
+					ast: p.currentStats == undefined ? 0.0 : p.currentStats.ast,
+				};
+			}
+
+			return {
+				...prevState,
+				aws,
+			};
+		});
+		return error;
+	};
+
+	const handleFormSubmit = async (event: FormEvent) => {
+		event.preventDefault();
+		setState(prevState => ({
+			...prevState,
+			saving: true,
+		}));
+
+		try {
+			await toWorker("main", "upsertAwards", state, awardsInitial);
+			realtimeUpdate([], helpers.leagueUrl(["history"]));
+		} catch (error) {
+			logEvent({
+				type: "error",
+				text: error.message,
+				saveToDb: false,
+				persistent: true,
+			});
+			setState(prevState => ({
+				...prevState,
+				saving: false,
+			}));
+		}
+	};
+	if (awards) {
+		return (
+			<form onSubmit={handleFormSubmit}>
+				<div className="row">
+					<div className="col-sm-3 col-6 form-group">
+						<label>Finals MVP</label>
+						<SelectReact
+							options={players}
+							player={awards["finalsMvp"]}
+							award="finalsMvp"
+							changing={handleChange.bind(this)}
+						/>
+					</div>
+					<div className="col-sm-3 col-6 form-group">
+						<label>MVP</label>
+						<SelectReact
+							options={players}
+							player={awards["mvp"]}
+							award="mvp"
+							changing={handleChange.bind(this)}
+						/>
+					</div>
+
+					<div className="col-sm-3 col-6 form-group">
+						<label>Rookie of the year</label>
+						<SelectReact
+							options={players}
+							player={awards["roy"]}
+							award="roy"
+							changing={handleChange.bind(this)}
+						/>
+					</div>
+					<div className="col-sm-3 col-6 form-group">
+						<label>Defensive Player of the year</label>
+						<SelectReact
+							options={players}
+							player={awards["dpoy"]}
+							award="dpoy"
+							changing={handleChange.bind(this)}
+						/>
+					</div>
+					<div className="col-sm-3 col-6 form-group">
+						<label>Six Man of the year</label>
+						<SelectReact
+							options={players}
+							player={awards["smoy"]}
+							award="dpoy"
+							changing={handleChange.bind(this)}
+						/>
+					</div>
+
+					<div className="col-sm-3 col-6 form-group">
+						<label>Most Improved player</label>
+						<SelectReact
+							options={players}
+							player={awards["mip"]}
+							award="mip"
+							changing={handleChange.bind(this)}
+						/>
+					</div>
+				</div>
+				<h1>Teams</h1>
+				<div className="row">
+					{awards["allLeague"].map((element: any, i: number) => {
+						const title = <h1 key={i}>{element.title} All NBA</h1>;
+						const teamSelect = element["players"].map(
+							(player: any, j: number) => {
+								return (
+									<div className="col form-group">
```

Comment:
Add `key={j}` - without that, you see a warning message in the JS console from React. Same on line 286.